backend/src/main/java/de/neuefische/CapStone/backend/api/CoffeeDevice.java

```diff
@@ -19,4 +19,5 @@
     private String uid;
     private String itemName;
     private boolean onOff;
+    private boolean dailyAction;
 }
```

Comment:
daily Action wird ein feature sein, in den Alarms bzw einmalige events (wie Kaffee Maschine einschalten), tägliche Events sind